includes/templates/class-amp-post-template.php

```diff
@@ -349,7 +349,7 @@ private function build_post_featured_image() {
 
 		list( $sanitized_html, $featured_scripts, $featured_styles ) = AMP_Content_Sanitizer::sanitize(
 			$featured_html,
-			array( 'AMP_Img_Sanitizer' => array() ),
+			amp_get_content_sanitizers( $post_id ),
```

Comment:
Should this pass a `WP_Post` instead of `$post_id`? It looks like `amp_get_content_sanitizers()` [expects a WP_Post](https://github.com/Automattic/amp-wp/blob/develop/includes/amp-helper-functions.php#L465) as the argument.